packages/farcaster-js-neynar/src/v2/NeynarV2APIClient.ts

```diff
@@ -135,18 +127,28 @@ export class NeynarV2APIClient {
   }
 
   /**
-   * Registers a Signer with an fid. See [Neynar documentation](https://docs.neynar.com/reference/register-app-fid)
+   * Creates and registers a Signer for an fid. See Neynar documentation[1](https://docs.neynar.com/reference/create-signer),[2](https://docs.neynar.com/reference/register-app-fid)
    * for more details.
    */
-  public async registerSigner(
-    signerUuid: string,
+  public async createAndRegisterSigner(
     fid: number,
     deadline: number,
```

Comment:
Do you think it would be helpful to set a default value for `deadline`, or just make it more confusing?

Reply:
Yeah that's a good idea, what do you think the default should be? I'm torn because I'd like it to be essentially infinite, but neynar doesn't provide a way to revoke the access for it.

Reply:
30 days seemed like a good compromise. I made it clear in the examples as well.